src/components/cylc/gscan/GScan.vue

```diff
@@ -61,18 +61,15 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
       >
         <v-card width="500px">
           <v-list>
-            <v-list-item>
+            <v-list-item
+              v-for="(items, title) in filters"
```

Comment:
I've gone back to this `for` loop as if you look at `$data.filters`, we have a couple of commented out fields that we might enable in future